### DISCUSSION TITLE
How do I use this in React?

### DISCUSSION THREAD

**Discussion by sinjs** [giscus/giscus#143] [Q&A] [answered]
Is there a React component so I can just use this in a Next.js / React application?

**Comment by laymonage** [comment]
Hey @sinmineryt!

You can see [an example in my blog](https://github.com/laymonage/base/blob/main/components/Giscus.tsx). I've also created a [react-giscus](https://github.com/giscus/react-giscus) component, which doesn't use a `<script>` tag. However, I haven't updated it to support the latest features I released last week.

**Reply by sinjs** [reply]
Thank you very much!

**Reply by deadcoder0904** [reply]
@laymonage Are you planning to maintain `react-giscus` so I can use that directly as it reduces lots of code?

**Reply by laymonage** [reply]
@deadcoder0904 I am planning to, but I also plan to fork https://github.com/TomokiMiyauci/utterances-component instead so that I can have a single codebase for React, Vue, and Svelte components.

**Reply by deadcoder0904** [reply]
@laymonage thank you for the link. this seems new but glad I found it so I'll probably use that too altho `giscus` for my blog :)

it's really great 🎉

**Comment by laymonage** [comment, marked as answer]
Note: the `react-giscus` component has been replaced by `@giscus/react`. See https://github.com/giscus/giscus-component.
